portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetSorting.testcase

```diff
@@ -25,30 +39,116 @@ definition {
 	}
 
 	@description = "LPS-185324. Confirm that the new default classification modal is displayed, the field is required."
-	@ignore = "true"
 	@priority = 4
 	test CanAssertFieldIsRequired {
+		task ("And adding a dataset view") {
+			FrontendDataSetAdmin.createDataSetView(
+				description = "View Description",
+				key_name = "View Test");
+		}
+
+		task ("When the user accesses the sorting tab") {
+			FrontendDataSetAdmin.goToTab(
+				dataSetViewName = "View Test",
+				tabName = "Sorting");
+		}
 
-		// TODO LPS-185324 CanAssertFieldIsRequired pending implementation
+		task ("And adds a new default sort, on "+" button or "New Default Sort" button") {
+			if (IsElementPresent(key_text = "No default sort was created yet", locator1 = "ObjectCustomViews#VIEW_BUILDER_EMPTY_STATE_TEXT")) {
+				ObjectCustomViews.addNewDefaultSort();
+			}
+			else {
+				Click(locator1 = "FrontendDataSetAdmin#BUTTON_ADD_SORT");
+			}
+		}
+
+		task ("And the new default sort modal is displayed") {
+			AssertElementPresent(
+				locator1 = "Portlet#MODAL_TITLE",
+				value1 = "New Default Sort");
+		}
 
+		task ("Then Field is required") {
+			AssertElementPresent(
+				key_text = "Field",
+				locator1 = "RaylifeApplicationModal#MODAL_DESCRIPTION");
+		}
 	}
 
 	@description = "LPS-185325. Confirm that the new default classification modal is displayed, the sorting is required."
-	@ignore = "true"
 	@priority = 4
 	test CanAssertSortingIsRequired {
+		task ("And adding a dataset view") {
+			FrontendDataSetAdmin.createDataSetView(
+				description = "View Description",
+				key_name = "View Test");
+		}
 
-		// TODO LPS-185325 CanAssertSortingIsRequired pending implementation
+		task ("When the user accesses the sorting tab") {
+			FrontendDataSetAdmin.goToTab(
+				dataSetViewName = "View Test",
+				tabName = "Sorting");
+		}
```

Comment:
Move `task ("And adding a dataset view") `and `task ("When the user accesses the sorting tab") `to set up